.github/workflows/daily-rhel-copr.yml

```diff
@@ -29,5 +44,13 @@ jobs:
           archive_cmd = make po-pull && make dist
           EOF
 
+      - name: Copy token
+        run: |
+          cp /home/github/copr-rhel-token copr-builder/copr-rhel-token
+
       - name: Run copr-builder
-        run: ./copr-builder -v -c copr-builder-rhel.conf -C /copr-rhel
+        run: |
+          sudo podman run -i --rm --privileged -v `pwd`/copr-builder:/copr-builder:ro ${{ env.CI_CONTAINER }} <<EOF
```

Comment:
Please use `-v ~/copr-rhel-token:...` instead. Also add `:Z` to the volume mount.

Reply:
I don't think I can mount a file like that?

Reply:
Based on my tests yes, you can :).